azext_edge/edge/providers/support/akri.py

```diff
@@ -28,6 +28,13 @@
 AKRI_SERVICE_LABEL = "service in (aio-akri-metrics)"
 AKRI_PREFIX = "aio-akri-"
 
+AKRI_AGENT_LABEL = "aio-akri-agent"
+AKRI_WEBHOOK_LABEL = "aio-akri-webhook-configuration"
+
+AKRI_NAME_LABEL = "app.kubernetes.io/name in (0)"
```

Comment:
Why not use a format string placeholder? in this case you can do `(0)` -> `{0}` or even `{}`.

Reply:
to add on - you can also move this out since the label structure _should_ be the same across services

if you want examples, look at how I use COLOR_STR_FORMAT in my pr